rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverDrawEventOnSubscribe.java

```diff
@@ -0,0 +1,40 @@
+package com.jakewharton.rxbinding.view;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.view.View;
+import android.view.ViewTreeObserver;
+import com.jakewharton.rxbinding.internal.MainThreadSubscription;
+import rx.Observable;
+import rx.Subscriber;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkUiThread;
+
+final class ViewTreeObserverDrawEventOnSubscribe
+    implements Observable.OnSubscribe<ViewTreeObserverDrawEvent> {
+  private final View view;
+
+  public ViewTreeObserverDrawEventOnSubscribe(View view) {
+    this.view = view;
+  }
+
+  @TargetApi(Build.VERSION_CODES.JELLY_BEAN) @Override public void call(final Subscriber<? super ViewTreeObserverDrawEvent> subscriber) {
```

Comment:
For `OnSubscribe` classes we put the `@TargetApi` annotation on the whole class declaration.